src/core/tests/rtti.cpp

```diff
@@ -2,9 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "common_test_utils/all_close_f.hpp"
 #include "common_test_utils/test_tools.hpp"
 #include "gtest/gtest.h"
+#include "ngraph/function.hpp"
```

Comment:
@olpipi will the file be refactored and when?

Reply:
Here we can use new namespaces

Reply:
Yes, I change it